SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-or-later

mes libc does not have locale support...

diff --git lib/sh/snprintf.c lib/sh/snprintf.c
index 7669576..747aeba 100644
--- lib/sh/snprintf.c
+++ lib/sh/snprintf.c
@@ -376,7 +376,7 @@ static void xfree __P((void *));
 	    if ((p)->flags & PF_STAR_P) \
 	      (p)->precision = GETARG (int)
 
-#if defined (HAVE_LOCALE_H)
+#if 0
 #  define GETLOCALEDATA(d, t, g) \
       do \
 	{ \
